yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/ShareSessionReceipt.java

```diff
@@ -0,0 +1,206 @@
+package com.yoti.api.client.spi.remote.call;
```

Comment:
maybe `client` instead of `call`? Or even `api`?

Reply:
following the current package structure, few things are shared and need to be cloned if I want to better structure the code. I opted to stick to the current (not very nice) structure rather than copying and pasting xxx classes